flake.nix

```diff
@@ -499,11 +513,19 @@
             '' + target.extraEnvs;
           });
 
-        fedimintd = pkg {
-          name = "fedimintd";
+        fedimint-pkgs = pkgsBuild {
+          name = "fedimint-pkgs";
+
+          pkgs = {
+            fedimintd = { };
+            fedimint-cli = { };
+            fedimint-tests = { };
```

Comment:
We might want to avoid test-only features being active in release builds eventually to reduce attack surface. But ok for now imo.